problems/2017-04-11-side-effects-may-include/README.md

```diff
@@ -0,0 +1,29 @@
+# Side Effects May Include
+
+Has this ever happened to you? You call a function twice in a row with the same parameters, and you get two different results!
+
+My friend, you're using an impure function. Impure functions aren't the worst thing in the world, sometimes you need them to modify state.
+
+In this challenge, you'll be refactoring bits of code that aren't as pure as they could be.
+
+## General Information
+
+We'll be using JavaScript run on NodeJS for this one. 
+
+There won't be a need for any external packages, aside from the ones you'll need to install to test your code.
```

Comment:
I might just leave this out. I think it's kinda confusing since you need to install dependencies from npm anyway.